localstripe/resources.py

```diff
@@ -244,6 +244,67 @@ def do_expand(path, obj):
         return obj
 
 
+class Balance(object):
+    object = 'balance'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        self.object = self.object
+        self.livemode = False
+        self.type = 'balance'
+        self.connect_reserved = []
+        self.available = {
+            'amount': 2000,
+            'currency': 'eur',
+            'source_types': {
+                'card': 2000
+            }
+        }
+        self.pending = {
+            'amount': 0,
+            'currency': 'eur',
+            'source_types': {
+                'card': 0
+            }
+        }
+
+        store[self.object] = self
+
+        schedule_webhook(Event('balance.available', self))
+
+    @classmethod
+    def _api_get(self):
```

Comment:
[Again](https://github.com/adrienverge/localstripe/pull/165#discussion_r556375194), the correct term is `_api_retrieve`.

Reply:
Oh, ok sorry I did not get what you meant